openstack/mrs/v1/cluster/requests.go

```diff
@@ -25,7 +20,7 @@ type CreateOpts struct {
 	SubnetName            string          `json:"subnet_name" required:"true"`
 	SecurityGroupsID      string          `json:"security_groups_id,omitempty"`
 	ClusterVersion        string          `json:"cluster_version" required:"true"`
-	ClusterType           int             `json:"cluster_type"`
+	ClusterType           int             `json:"cluster_type,omitempty"`
```

Comment:
ClusterType can be 0, using `omitempty` will disallow setting it to 0 explicitly